spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    association :project
+
+    trait :done_project do
```

Comment:
このtrait名だと関連しているProjectが完了しているように思えますが、完了しているのは単体のtaskです。
誤解を生まないようにprojectという単語は外してしまった方が良いでしょう。
```suggestion
    trait :done do
```

Reply:
a6169ff で修正しました。